tests/ci/docker_images/linux-x86/build_images.sh

```diff
@@ -17,3 +21,9 @@ docker build -t fedora-31:clang-9x fedora-31_clang-9x
 docker build -t integration:s2n s2n_integration_clang-9x
 docker build -t ubuntu-20.04:gcc-9x ubuntu-20.04_gcc-9x
 docker build -t ubuntu-20.04:clang-10x ubuntu-20.04_clang-10x
+
+#############################################
+# Build images defined in other GitHub repo #
```

Comment:
Can't hurt to provide the actual repo name here:

```
# Build images defined in aws-lc-verification Github repo #
```

Reply:
Updated the comment.